internal/app/wwctl/container/list/main_test.go

```diff
@@ -83,13 +71,12 @@ WW_INTERNAL: 0
 				stdoutC <- buf.String()
 			}()
 			stdoutW.Close()
-
 			stdout := <-stdoutC
-			assert.NotEmpty(t, stdout, "os.stdout should not be empty")
-			if !strings.Contains(stdout, tt.stdout) {
-				t.Errorf("Got wrong output, got:\n '%s'\n, but want:\n '%s'\n", stdout, tt.stdout)
-				t.FailNow()
-			}
+			assert.Equal(t, tt.output, stdout)
+			assert.Equal(t,
+				strings.ReplaceAll(strings.TrimSpace(tt.output), " ", ""),
+				strings.ReplaceAll(strings.TrimSpace(stdout), " ", ""))
```

Comment:
I don't really like this pattern of testing output with all whitespace removed. But I don't know that I'd stand in the way of a pr for it at this stage: we need the CLI to be better about whitespace.

Reply:
I am trying to make the output equal with whitespaces, but this seems a bit more robust.